lesson_10/test_homework10.py

```diff
@@ -0,0 +1,40 @@
+"""Tests for function log_event."""
+import logging
+from unittest.mock import patch
+
+from homework_10_log_event import log_event
+
+
+def test_log_event():
+    """Test the log_event function with various statuses."""
+    with patch('logging.getLogger') as mock_get_logger:
+        logger = mock_get_logger.return_value  # Create mock logger
+
+        # Check status "success"
+        log_event('user1', 'success')
+        logger.info.assert_called_with(
+            'Login event - Username: user1, Status: success',
+        )
+
+        # Check status "expired"
+        log_event('user2', 'expired')
+        logger.warning.assert_called_with(
+            'Login event - Username: user2, Status: expired',
+        )
+
+        # Check status "failed"
+        log_event('user3', 'failed')
+        logger.error.assert_called_with(
+            'Login event - Username: user3, Status: failed',
+        )
+
+        # Check unknown status
+        log_event('user4', 'unknown')
+        logger.error.assert_called_with(
+            'Login event - Username: user4, Status: unknown',
+        )
+
+
+if __name__ == '__main__':
+    test_log_event()
+    logging.info('Tests passed!')
```

Comment:
No no no there should be four pytest test cases. Then you could run it as `pytest -sv <name_of your_test_module>`. No sense to put all testing within one TC and no sense to add main block. Also please switch mocking to https://pytest-mock.readthedocs.io/en/latest/